x-pack/plugins/infra/public/pages/metrics/inventory_view/components/ml/anomaly_detection/job_setup_screen.tsx

```diff
@@ -45,8 +58,10 @@ export const JobSetupScreen = (props: Props) => {
   const [filterQuery, setFilterQuery] = useState<string>('');
   const trackMetric = useUiTracker({ app: 'infra_metrics' });
   const { createDerivedIndexPattern } = useSourceContext();
+  const { kibanaVersion, isCloudEnv, isServerlessEnv } = useContext(KibanaEnvironmentContext);
+  const { euiTheme } = useEuiTheme();
 
-  const indicies = h.sourceConfiguration.indices;
+  const indices = h.sourceConfiguration.indices;
```

Comment:
Can you improve this `h` variable name to something meaningful?

Reply:
This is not something introduced in this PR ( I just fixed a typo: `indicies` to `indices` here) there are a lot of improvements we can make here but I tried to keep the focus. I will rename it to `host` so it's clear it's coming from the hosts context 👍

Reply:
I renamed both so now `h` is `host` and `k` is `kubernetes`

Reply:
Thank you